### ISSUE TITLE
give users the ability to delete their quotes

### ISSUE THREAD

**Issue by k-e-l-p** [FrozenPigs/Taigabot#61]
Why was this feature removed in the first place? Seems to be a bit silly since anyone can just come into the channel and spam up any user with shitty quotes

**Comment by FrozenPigs** [comment]
>Why was this feature removed in the first place? 

Wrong, the feature was always global admin only, until I made it not for a while. Then I got bored of people removing funny quotes just because they didn't like them, so I made it global admin only again.

**Comment by ineeee** [comment]
it was removed because you fuckers in homescreen kept deleting real quotes you didnt want to be seen
